lang/go/go.talon

```diff
@@ -194,3 +269,19 @@ swipe [<user.text>] [over]:
     key("right")
     insert(", ")
     insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
+
+type <user.text> struct:
+    insert("type ")
+    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
+    insert(" struct {")
+    key("Enter")
```

Comment:
Is this relying on editor inserting a `}`?  Worth looking to see how we do this for eg function definitions / `for` loops / `if` statements in other C-style languages